src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -50,13 +50,13 @@ public function copy(string $originFile, string $targetFile, bool $overwriteNewe
 
         if ($doCopy) {
             // https://bugs.php.net/64634
-            if (false === $source = @fopen($originFile, 'r')) {
-                throw new IOException(sprintf('Failed to copy "%s" to "%s" because source file could not be opened for reading.', $originFile, $targetFile), 0, null, $originFile);
+            if (!$source = self::box('fopen', $originFile, 'r')) {
+                throw new IOException(sprintf('Failed to copy "%s" to "%s" because source file could not be opened for reading: ', $originFile, $targetFile).self::$lastError, 0, null, $originFile);
```

Comment:
Why not use the `sprintf` instead of concatenate `self::$lastError`

Reply:
because fabbot would complain about missing double quotes around, which we don't want